docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
+
+In User Sync, you can use user groups or Product Configurations in the mapping from directory groups.  So users can be added to or removed from user groups as well as PCs.  You can't create new user groups or product configurations however; that must be done in the Admin Console.
+
+### In the examples in the user manual I see that each directory group is mapped to exactly one Adobe group; is it possible to have 1 AD group map to multiple product configurations?
 
-**Q: In the documentation I see that there is a 1:1 match (1 AD group = 1 product 
-configuration); is it possible to have 1 AD group and multiple product 
-configurations?** How about 1 AD group vs many AC groups?<br/>
-A: Most of the examples show just a single Adobe user group or PC, but the mapping can be one to many.  Simply list all the user groups or PCs, one per line, with a leading "-" (and indented to the proper level) on each as per YML list format.
+Most of the examples show just a single Adobe user group or PC, but the mapping can be one to many.  Simply list all the user groups or PCs, one per line, with a leading "-" (and indented to the proper level) on each as per YML list format.
 
-**Q: I understand the user sync tool would do a GET users from the org (or product 
+### I understand the user sync tool would do a GET users from the org (or product 
 configuration?) in order to compare with the extract of users from AD. It is 
-known the limitation of number of requests per minute to our server.  Is this 
-a problem?**  For example,  the back-off can stop the activity to query or update 
+known the limitation of number of requests per minute to Adobe's server.  Is this 
+a problem?
+
+For example,  the back-off can stop the activity to query or update 
 in X number of tries. This can leave a partially updated list of users or a 
-partial query.<br/>
-A: No, User sync handles throttling and retries so that throttling may slow 
+partial query.
+
+No, User sync handles throttling and retries so that throttling may slow 
 down the overall user sync process, but there is no problem caused by throttling 
 and user sync will properly complete all operations.
  
-**Q:  Is there a local list of users created/updated (on the user sync side) 
+###  Is there a local list of users created/updated (on the user sync side) 
 in order not to disturb our server every time for a list of users from the 
-org/product configuration?**<br/>
-A: No, User sync always queries the Adobe user management systems to get 
+org/product configuration?
+
+No, User sync always queries the Adobe user management systems to get 
 current information when it is run.
 
-**Q: I’d like to know more on how the user sync tool deals with the throttling 
-in general.**<br/>
-A: The Adobe systems protect themselves from overload by tracking the incoming 
+### I’d like to know more on how the user sync tool deals with the throttling 
```

Comment:
broken question.  Also it doesn't need to be a separate question; you can just merge this answer with the answer to the original throttling question, as an additional paragraph.